### PR TITLE
Cs/layout

### PR DESCRIPTION
**Pull request by monstaro**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds routing for collections page and POST request

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [X] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/javascript/readingList/readingList.jsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const FilterText = ({ selectedTags, query, value }) => {
 export class ReadingList extends Component {
   constructor(props) {
     super(props);
+    console.log(props);
```
Comment (nkeller1) [inline app/javascript/readingList/readingList.jsx:37]: Console log here to take out on next PR

File: app/javascript/tagCollections/tagCollections.jsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    setTimeout(() => {
+      t.setState({ archiving: false });
+    }, 1000);
+  };
+
+  postCollection = e => {
+    e.preventDefault();
+    const params = {
+      name: e.target.firstChild.value,
+      tag_list: 'javascript',
```
Comment (nkeller1) [inline app/javascript/tagCollections/tagCollections.jsx:118]: We will need this to be an actual form to submit a tag but glad its working,. Was it the preventdefualt that did it?

File: app/javascript/tagCollections/tagCollections.jsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+    };
+    console.log(JSON.stringify(params));
```
Comment (nkeller1) [inline app/javascript/tagCollections/tagCollections.jsx:120]: console log to remove

File: app/javascript/readingList/readingList.jsx (hunk 1 above)
Comment (monstaro) [reply app/javascript/readingList/readingList.jsx:37]: Thanks for pointing that out!

File: app/javascript/tagCollections/tagCollections.jsx [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+    fetch('/tagcollections', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'X-CSRF-Token': window.csrfToken,
```
Comment (nkeller1) [inline app/javascript/tagCollections/tagCollections.jsx:126]: nevermind. guess this is what did it!

**Review by nkeller1 — APPROVED** [review]
Few console logs to remove and need to add the second form field but glad its creating tagcollections!

**Pull request merged**